### ISSUE TITLE
Cannot pull container images into VCH with openssh-server installed

### ISSUE THREAD

**Issue by jplaa** [vmware/vic#8153]
Hello,

Every time I try to import a container image with SSH installed from the Registry to the VCH host I get the error:
Failed to write to image store: [POST /storage/{store_name}] [5000] WriteImage default &{Code: 500 Message:Failed to add disk 'scsi0:0' .}

We are currently using VIC 1.3.

Any help with this would be greatly appreciated. 

Thanks.

**Comment by lcastellano** [comment]
I am not sure I understand this issue. Has the openssh-server been manually installed on the VCH (with tdnf, yum, ..)? Is the problem that after installing and running openssh-server you cannot successfully pull container images from "any" registry? In addition can you please provide the VIC log files (https//<VCH-ADDRESS>:2378).

**Comment by jplaa** [comment]
I tested this again and the problem might not be related to openssh-server but I will let you know what I did. 
I created a new VCH and tried to pull an image from the registry then I got the following error:
Failed to write to image store: [post /storage/{store_name}] [500] WriteImage default &{Code:500 Message:ServerFaultCode: XML element depth exceeds configured maximum 100

After that, I tried to import the same image and got a different error:
Failed to write image store: [POST /storage/{store_name}] [500] WriteImage default &{Code:500 Message:Invalid configuration for device '0'.}

Then I was able to import a different image successfully.  

[vic-logs.zip](https://github.com/vmware/vic/files/2220730/vic-logs.zip)

**Comment by lcastellano** [comment]
It looks like this problem is related to the number of layers in the container image. I was able to reproduce it with an image containing approximately 90 layers. I am still investigating how to recover from this situation once the error occurred.

**Comment by jplaa** [comment]
Thank you. Do you think then this is a limitation of the VCH? Do you think  I will not be able to pull container images with too many layers?

**Comment by jplaa** [comment]
Yes, it is an image that we built. We will try to reduce the number of layers in the image and then it should work. We did not know that the number of layers could be a limitation so thank you very much for your help!

**Comment by stuclem** [comment]
Opened https://github.com/vmware/vic-product/issues/1974 for the doc update
